packages/ag-charts-enterprise/src/features/annotations/annotations.ts

```diff
@@ -493,27 +473,68 @@ export class Annotations extends _ModuleSupport.BaseModuleInstance implements _M
 
             // Services
             ctx.annotationManager.addListener('restore-annotations', this.onRestoreAnnotations.bind(this)),
-            ctx.toolbarManager.addListener('button-pressed', this.onToolbarButtonPress.bind(this)),
-            ctx.toolbarManager.addListener('button-moved', this.onToolbarButtonMoved.bind(this)),
-            ctx.toolbarManager.addListener('group-moved', this.onToolbarGroupMoved.bind(this)),
-            ctx.toolbarManager.addListener('cancelled', this.onToolbarCancelled.bind(this)),
             ctx.layoutManager.addListener('layout:complete', this.onLayoutComplete.bind(this)),
             ctx.updateService.addListener('pre-scene-render', this.onPreRender.bind(this)),
             ctx.zoomManager.addListener('zoom-change', () => this.onZoomChange()),
 
+            // Toolbar
```

Comment:
Annotation toolbar events are handled here, mostly to cancel creating, reset and update the colors.